src/Umbraco.Infrastructure/Migrations/Upgrade/V_9_4_0/AddScheduledPublishingLock.cs

```diff
@@ -0,0 +1,15 @@
+using Umbraco.Cms.Infrastructure.Persistence.Dtos;
+
+namespace Umbraco.Cms.Infrastructure.Migrations.Upgrade.V_9_4_0
```

Comment:
```suggestion
namespace Umbraco.Cms.Infrastructure.Migrations.Upgrade.V_10_0_0
```

Reply:
This migration was added in 9.4.0 though, the commit introducing it into v10 is a cherrypick from v9